.github/workflows/build-deploy-docs.yml

```diff
@@ -15,6 +15,20 @@ jobs:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
 
+      - name: Run link check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'no'
+          use-verbose-mode: 'yes'
+          folder-path: 'docs/src'
+          file-path: './README.md'
+          config-file: './.github/files/markdown.links.config.json'
+
+      - name: Markdown lint
+        uses: DavidAnson/markdownlint-cli2-action@v14
+        with:
+          config: './.github/files/.markdownlint-cli2.jsonc'
```

Comment:
don't forget to rename this file.  Possible filename:   `files/markdownlint-cli2.jsonc` (without the dot)

Reply:
Already renamed to `martin.markdownlint-cli2.jsonc`.    
Seems there's some weird restriction on configuration file naming.
```shell
zyj@junpc:~/repos/martin$ markdownlint-cli2 --config ".github/files/markdownlint-cli2.jsonc" --fix
markdownlint-cli2 v0.11.0 (markdownlint v0.32.1)
Error: Configuration file "/home/zhangyijun/repos/martin/.github/files/markdownlint-cli2.jsonc" is unrecognized; its name should be (or end with) one of the supported types (e.g., ".markdownlint.json" or "example.markdownlint-cli2.jsonc").
    at readOptionsOrConfig (/usr/local/lib/node_modules/markdownlint-cli2/markdownlint-cli2.js:191:11)
    at main (/usr/local/lib/node_modules/markdownlint-cli2/markdownlint-cli2.js:935:13)
    at /usr/local/lib/node_modules/markdownlint-cli2/markdownlint-cli2.js:1045:32
    at run (/usr/local/lib/node_modules/markdownlint-cli2/markdownlint-cli2.js:1050:5)
    at Object.<anonymous> (/usr/local/lib/node_modules/markdownlint-cli2/markdownlint-cli2.js:1062:3)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
```